src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,21 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
+        OrderService orderService = new OrderServiceImpl();
         User user;
         try {
-            user = authenticationService.login("bob", "1234");
+            user = authenticationService.login("bob", "1254");
+            orderService.completeOrder(user.getUserId());
         } catch (AuthenticationException e) {
-            e.printStackTrace();
-            return;
+            logger.error("Username or password are incorrect" + e);
```

Comment:
to make it more clear
```suggestion
            logger.error("Login failed. Username or password are incorrect" + e);
```